Source/UI/CommonUI/UIStyle/CKOUIStyle+UIFont.swift

```diff
@@ -0,0 +1,37 @@
+import UIKit
+
+enum CKOUIStyle {}
+
+extension CKOUIStyle {
+
+    enum Font {
+        enum Title {
+            static let title2 = CKOUIStyle.font(size: 24, weight: .medium)
+            static let headline = CKOUIStyle.font(size: 17, weight: .semibold)
```

Comment:
I don't think we need to have the "Title/Body/Input/Action" enums.

I would say we should flatten it up to keep it simpler not force extra abstraction layers.

For reference Apple Typography doesn't separate Title, Body, Action fonts either. And if we do for our theming, we can have bodyLarge, body, bodyPlus , which would further save you from having to use `default`.